zds/settings/prod.py

```diff
@@ -63,7 +61,8 @@
 MEDIA_ROOT = Path("/opt/zds/data/media")
 
 STATIC_ROOT = Path("/opt/zds/data/static")
-STATICFILES_STORAGE = "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
+
+STORAGES = {"staticfiles": {"BACKEND": "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"}}
```

Comment:
En essayant de déployer avec Vagrant, ça plante parce que ça ne trouve pas `STORAGE["default"]`.  D'après [la doc](https://docs.djangoproject.com/en/4.2/ref/settings/#storages), il faut aussi définir la clé `default` : 
```suggestion
STORAGES = {
    "default": {
        "BACKEND": "django.core.files.storage.FileSystemStorage",
    },
    "staticfiles": {
        "BACKEND": "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
    },
}
```